Stratis.Bitcoin/Features/BlockStore/LoopSteps/BlockStoreInnerStepReadBlocks.cs

```diff
@@ -3,27 +3,27 @@
 using Stratis.Bitcoin.BlockPulling;
 using System;
 using System.Collections.Generic;
-using System.Linq; 
+using System.Linq;
 using System.Threading.Tasks;
 
 namespace Stratis.Bitcoin.Features.BlockStore.LoopSteps
 {
     /// <summary>
-    /// Downloads blocks from the BlockPuller removes block from the DownloadStack.
+    /// Reads blocks from the <see cref="BlockPuller"/> and removes block from the <see cref="BlockStoreInnerStepContext.DownloadStack"/>.
     /// <para>
     /// If the block exists in the puller add the the downloaded block to the store to
     /// push to the repository. If the <see cref="BlockStoreLoop.MaxInsertBlockSize"/> has been reached
     /// push the blocks in the context's Store to the repository.
     /// </para> 
     /// <para>
-    /// When the download stack is empty return a Break() result causing the BlockStoreLoop to
+    /// When the download stack is empty return a Break() result causing the <see cref="BlockStoreLoop"/> to
```

Comment:
Break is not a method, I event can't see any Break in the code, do you mean InnerStepResult.Stop?